easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -193,9 +212,8 @@ def test_step(self):
                 abs_pylibdirs = [os.path.join(testinstalldir, pylibdir) for pylibdir in self.all_pylibdirs]
                 extrapath = "export PYTHONPATH=%s && " % os.pathsep.join(abs_pylibdirs + ['$PYTHONPATH'])
 
-                tup = (extrapath, self.cfg['preinstallopts'], testinstalldir, self.cfg['installopts'])
-                cmd = "%s%s python setup.py install --prefix=%s %s" % tup
-                run_cmd(cmd, log_all=True, simple=True)
+                tup = (extrapath, self.cfg['preinstallopts'], self.install_cmd, testinstalldir, self.cfg['installopts'])
+                run_cmd("%s%s %s --prefix=%s %s" % tup, log_all=True, simple=True)
```

Comment:
I could, but there's already a space at the end in `extrapath`; maybe removed it there and adding it here is clearer